zeppelin-web/src/index.js

```diff
@@ -12,6 +12,7 @@
  * limitations under the License.
  */
 
+import 'angular-viewport-watch/angular-viewport-watch.js'
```

Comment:
Thanks for update! It would be nicer to move import stmt into `app.js` where it is used.